ogip/spec.py

```diff
@@ -80,13 +80,38 @@ def from_file_name_osa(fn):
 
     @staticmethod
     def from_file_name_normal(fn):
+
         f = fits.open(fn)
+        if 'RATE' in f['SPECTRUM'].data.names:
+            rate = f['SPECTRUM'].data['RATE']
+        elif 'COUNTS' in f['SPECTRUM'].data.names:
+            rate = f['SPECTRUM'].data['COUNTS'] / f['SPECTRUM'].header['EXPOSURE']
+        else:
+            rate = None
+
+        if 'STAT_ERR' in f['SPECTRUM'].data.names:
+            stat_err = f['SPECTRUM'].data['STAT_ERR']
+        elif 'POISSERR' in f['SPECTRUM'].header and f['SPECTRUM'].header['POISSERR']:
+            if 'COUNTS' in f['SPECTRUM'].data.names:
+                stat_err = np.sqrt(f['SPECTRUM'].data['COUNTS']) / f['SPECTRUM'].header['EXPOSURE']
+            elif 'RATE' in f['SPECTRUM'].data.names:
```

Comment:
is that anticipated in the standard? POISSERR with RATE columns? EXPOSURE might be somehow corrected (e.g. deadtime) and RATE*EXPOSURE can be non-int. Interpretation might be complex.

If it is anticipated, could you cite it in a comment for clarity?

Reply:
If the POISSERR keywork is set, it is meant that uncertainty is the square root of counts. As the output is only using rate, a correct assumption is that RATE=COUNTS/EXPOSURE and STAT_ERR = sqrt(COUNTS)/EXPOSURE